libdeno/api.cc

```diff
@@ -43,8 +43,7 @@ Deno* deno_new(deno_buf snapshot, deno_buf shared, deno_recv_cb cb) {
 }
 
 Deno* deno_new_snapshotter(deno_buf shared, deno_recv_cb cb,
-                           const char* js_filename, const char* js_source,
-                           const char* source_map) {
+                           const char* js_filename, const char* js_source) {
```

Comment:
This is a welcome change. `source_map` shouldn't have been part of the API, since libdeno did nothing but store it.